catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/operation/impl/CreateStorageResponseImpl.java

```diff
@@ -60,4 +62,16 @@ public CreateStorageResponseImpl(CreateStorageRequest request,
     public StorageRequest getStorageRequest() {
         return request;
     }
+
+    @Override
+    public String toString() {
```

Comment:
❗️ Now that you've fixed this issue, `ExportCommandSpec.groovy` ignored test should be re-enabled. 
Find the line `// TODO - Ignored until DDF-3123 has been addressed` and remove it after re-enabling the test.